docs/docsite/rst/dev_guide/developing_modules_in_groups.rst

```diff
@@ -104,6 +103,54 @@ If you need further advice, consider join the ``#ansible-devel`` IRC channel (se
 
 We have a ``ansibullbot`` helper that comments on GitHub Issues and PRs which should highlight important information.
 
+Example: Adding a group of modules
+==================================
+
+For example, to add a group of modules for ``myprod`` product, under a new ``newtech`` category , you would follow these steps:
+
+#. Create the new category directory in ``lib/ansible/modules`` and add an empty ``__init__.py`` file:
+
+    .. code-block:: bash
+
+      $ ls lib/ansible/modules/newtech
```

Comment:
If we need to do this, I'd just show a directory tree with the `__init__.py` files https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#directory-layout

so this would be in a single code-block